SDK/AppCenter/Microsoft.AppCenter.Shared/AppCenter.cs

```diff
@@ -11,50 +13,71 @@ namespace Microsoft.AppCenter
     /// </summary>
     public partial class AppCenter
     {
-        // Gets the first instance of an app secret corresponding to the given platform name, or returns the string 
+        const char SecretDelimiter = ';';
```

Comment:
These 4 const can be `private`, right?

Reply:
I think the default visibility is private here, and if you add private then Xamarin Studio will suggest you to remove it.

Reply:
![image](https://user-images.githubusercontent.com/4079255/67602096-c4907780-f72a-11e9-9dc1-70e8095f8ee1.png)
I don't think the default access modifier of C# const is `private`.  And you can see that VS for Mac does allow me to add `private`  here.

Reply:
https://stackoverflow.com/a/3763638/598520